README.md

```diff
@@ -205,29 +193,27 @@ Example:
 
 > Mention all of the collaborators of this project.
```

Comment:
- Please, let's also remove this placeholder text as this is here to guide you on how to add content to the readme as you should remove them after making the necessary changes.